contributing/contributing-code.md

```diff
@@ -90,8 +100,10 @@ Once you have found a reviewer, make sure to @-mention them on any issues / PRs
   - [Push your changes](./getting-started-e2e.md#push-your-changes-to-your-github-fork)
   - [Create a Pull Request (PR)](./getting-started-e2e.md#send-a-pull-request-ie-request-a-code-review).
   - Make sure the presubmit checks shown on your PR on GitHub passes (e.g. no lint and type check errors, tests are passing).
+  - Add reviewers to your PR that will fulfill the requirements of code review and approval documented in the [Code review and approval](#code-review-and-approval) section.  (Your reviewer can help with this.)
```

Comment:
Is it now possible to add reviewers to PRs for folk w/o write access?

Reply:
i believe it is not, as far as i know

Reply:
Yes, unfortunately they can't.  That's why I added "your reviewer can help with this."